### DISCUSSION TITLE
Host interview scheduling flow questions

### DISCUSSION THREAD

**Discussion by sanya301** [hackforla/HomeUniteUs#427] [Q&A]
1. How do co-ordinators schedule host interviews in terms of finding a common time for 3+ people?
    a. Do guests and hosts submit availability and co-ordinators tries to find a common time?
    b. Do hosts submit availability first? Or guests? Or is it ad hoc similar to whentomeet where everyone fills their availability?
    c. Are guests allowed to see host schedules and vice-versa? (this could be a schedule specifically entered for the purposes of the host interview, this does not imply ability to see the entire schedule)

@ExperimentsInHonesty @rossbb @JRHutson

**Comment by rossbb** [comment]
We have limited info on the calendaring micro-service. Here is what we know/assume so far:
1. Coordinators set their availability and guests and hosts select available slots from that. 
2. Each party selects "all" available slots from the first list, narrowing the availability as a sort of pyramid.

**Comment by rossbb** [comment]
For reference, here is the calendaring micro-service that has been built:
https://github.com/hackforla/calendaring-service

**Comment by rpradheap** [comment]
I assume that coordinators support multiple guests at a time. 
1.So, if one guest blocks available slots but the guest doesn't choose any slot - what will happen to the process. Does the guest need to find new slots  again?. 
2.Also, wouldn't the coordinator lose the availability- So is the available slots reserved only for a limited amount of time?
